tests/gold_tests/headers/general-connection-failure-502.gold

```diff
@@ -1,10 +1,10 @@
-HTTP/1.1 502 connect failed
-Connection: keep-alive
-Cache-Control: no-store
-Content-Type: text/html
-Content-Language: en
-Content-Length: 247
-
+HTTP/1.1 502 connect failed
+Connection: keep-alive
+Cache-Control: no-store
+Content-Type: text/html
+Content-Language: en
+Content-Length: 247
+

```

Comment:
autest currently normalizes white-space line-ending characters, effectively making carriage-return, line-feed just a line-feed, and then comparing output.

This means it is currently not possible to write correct tests, as false positives result when there are errors in the line endings.

Nevertheless, I am taking the opportunity to make this test more accurate now.